src/index.html

```diff
@@ -63,13 +63,11 @@
     <main>
       <section class="catalog page__catalog">
         <div class="card catalog__card" data-qa="card">
-          <div class="card__image-block">
-            <img
-            class="card-image"
-            src="images/imac.jpeg"
-            alt="iMac"
-            >
-          </div>
+          <img
+          class="card__image"
+          src="images/imac.jpeg"
+          alt="iMac"
+          >
```

Comment:
```suggestion
          <img
            class="card__image"
            src="images/imac.jpeg"
            alt="iMac"
          >
```
fix formatting everywhere